### PR TITLE
Staggered boxes

### PR DESCRIPTION
**Pull request by BNewing**
This adds in the staggered box component. 

I still need to do the full browser testing. I've done a quick scan of the major browsers with Browser stack, and seen a couple of issues with Internet Explorer, but nothing that makes it unusable - and the same issues affect other components, so it might be more to do with the layout. 

Any further changes I'll make will be CSS, which shouldn't break the block (we should be able to select 'Block recovery' and keep any content). So I'm putting this one through now so it can go up on the site for Josh, so he can start to add in content. 

After this goes up, I'll then do more thorough cross-browser testing, and go through with Tony to look at margins and paddings separately.

### CODE REVIEW

File: build/mojblocks.min.js [hunk 1]
```diff
@@ -2589,9 +2589,9 @@ if (mojblocksAccordion) {
 /*! no static exports found */
 /***/ (function(module, exports, __webpack_require__) {
 
-__webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/src/frontend.js */"./src/frontend.js");
-__webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/style.scss */"./style.scss");
-module.exports = __webpack_require__(/*! /Users/adam/Dev/moj/wp-jotw-public/web/app/plugins/wp-moj-blocks/style-gutenburg.scss */"./style-gutenburg.scss");
+__webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/src/frontend.js */"./src/frontend.js");
+__webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/style.scss */"./style.scss");
+module.exports = __webpack_require__(/*! /Users/beverleynewing/sites/wp-jotw-public/web/app/plugins/wp-moj-blocks/style-gutenburg.scss */"./style-gutenburg.scss");
```
Comment (RobLoweMOJ) [inline build/mojblocks.min.js:2595]: Not sure whats happening with this file. We should avoid having local stuff in it. Lets have a discussion.

Comment (RobLoweMOJ) [reply build/mojblocks.min.js:2595]: Also its not minfied

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "@wordpress/i18n": "^3.14.0",
     "@wordpress/scripts": "^12.1.1",
     "copy-webpack-plugin": "^6.0.1",
+    "govuk-frontend": "^3.7.0",
     "imagemin-mozjpeg": "^9.0.0",
```
Comment (RobLoweMOJ) [inline package.json:21]: If its in dependences do we need it in devdependences?

File: src/06-staggered-boxes/editor.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Plugin Name: MOJ Blocks
+Block: Accordion (Gutenberg editor styles)
+Text Domain: mojblocks
+
+**********************************************************************************************/
+
+// https://design-system.service.gov.uk/components/accordion/
+@import "node_modules/govuk-frontend/govuk/components/accordion/accordion";
+
```
Comment (brown-a2) [inline src/06-staggered-boxes/editor.scss:12]: prob don't need this one anymore?

File: package.json (hunk 2 above)
Comment (BNewing) [reply package.json:21]: Ah - we do not! I'll remove it :)

File: src/06-staggered-boxes/editor.scss (hunk 3 above)
Comment (BNewing) [reply src/06-staggered-boxes/editor.scss:12]: Ok! I'll take this out too :)

File: build/mojblocks.min.js (hunk 1 above)
Comment (BNewing) [reply build/mojblocks.min.js:2595]: Oh that's strange o.0 how come it's working like that?

File: package.json (hunk 2 above)
Comment (BNewing) [reply package.json]: Now removed :)

File: src/06-staggered-boxes/editor.scss (hunk 3 above)
Comment (BNewing) [reply src/06-staggered-boxes/editor.scss]: Also now removed :)

**Pull request merged**